pom.xml

```diff
@@ -19,6 +19,8 @@
 
     <dep.plugin.jacoco.version>0.8.3</dep.plugin.jacoco.version>
     <dep.plugin.javadoc.version>3.0.1</dep.plugin.javadoc.version>
+    <dep.hubspot-immutables.version>1.9</dep.hubspot-immutables.version>
```

Comment:
Not sure about adding immutables into this. In the past, non-HS users have disliked any other dependencies as it makes it difficult to manage version conflicts. You could manually create a builder object for this instead.

Reply:
good point.